packages/build/package.json

```diff
@@ -113,6 +112,7 @@
     "get-port": "^5.1.1",
     "get-stream": "^5.2.0",
     "has-ansi": "^4.0.0",
+    "is-ci": "^3.0.0",
```

Comment:
Is this needed as a `devDependencies`?

Reply:
Yeah, in tests, specifically:
- https://github.com/netlify/build/blob/88852c4a584a60170084b69dcf21c878867cc6e8/packages/build/tests/log/tests.js#L6
- https://github.com/netlify/build/blob/88852c4a584a60170084b69dcf21c878867cc6e8/packages/config/tests/cli/tests.js#L8

Reply:
Oh I missed that! :+1: